app/src/main/kotlin/compiler/lowlevel/storage/DisplayStorage.kt

```diff
@@ -5,6 +5,9 @@ import java.io.PrintWriter
 
 class DisplayStorage(val program: Program) {
     fun writeAsm(output: PrintWriter) {
-        TODO() // Reserve memory for the display with appropriate maximum depth
+        output.write("extern malloc")
```

Comment:
This is supposed to be in the `.bss` section, at the label used by `DefaultFunctionDetailsGenerator`.

Reply:
How should we call this label?